### PR TITLE
Adding core response objects for jackson deserialization

### PR DESCRIPTION
**Pull request by dafty**

### CODE REVIEW

File: src/main/java/io/ark/core/network/response/v1/Account.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import lombok.*;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@ToString(of = {"address"})
+@EqualsAndHashCode(of = {"address"})
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Account implements NodeResponse {
+
+    @Getter
```
Comment (okjintao) [inline src/main/java/io/ark/core/network/response/v1/Account.java]: Can we move these to the top of classes if they will be on each field?

File: src/main/java/io/ark/core/network/response/v1/Account.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    private int unconfirmedSignature;
+
+    @Getter
+    @Setter
+    private int secondSignature;
+
+    @Getter
+    @Setter
+    private String secondPublicKey;
+
```
Comment (okjintao) [inline src/main/java/io/ark/core/network/response/v1/Account.java]: Are we missing multisig lists?

**Review by okjintao — APPROVED** [review]
Let's use these classes for response mapping. I have some very similar classes here:

https://github.com/ArkEcosystem/ark-java/blob/jackson-response/src/main/java/io/ark/core/model/Delegate.java

Instead, let's use the classes on `model` and use them as the functional classes while these can be used to map responses.

File: src/main/java/io/ark/core/network/response/v1/Account.java (hunk 2 above)
Comment (dafty) [reply src/main/java/io/ark/core/network/response/v1/Account.java]: Multisig isn't implemented in v1, so we can omit them for now.

File: src/main/java/io/ark/core/network/response/v1/Account.java (hunk 1 above)
Comment (dafty) [reply src/main/java/io/ark/core/network/response/v1/Account.java]: Good spot.

**Pull request merged**